active-fedora.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
   s.add_dependency "active-triples", '~> 0.11.0'
   s.add_dependency "deprecation"
   s.add_dependency "ldp", '~> 0.6.0'
+  s.add_dependency "ruby-progressbar", '~> 1.0'
```

Comment:
I got some positive feedback on progress-bar in slack from jcoyne before adding it. I've found it invaluable in our local version of the reindex task. 

ruby-progress bar gives you a lot more than simulating it with a `puts`, like automatic calculation of percentage complete and ETA. i'm not really interested in simulating that with puts, when ruby-progressbar is already a mature, robust, and reliable dependency that's smoothed out the edge cases, and is already likely a dependency (albeit dev-group only) in the majority of hydra-using apps. I've used ruby-progressbar a lot in the past, and trust it to remain backwards compat in 1.x, it always has. 

But I can remove the progress bar feature if there isn't interest in it in the community. I added it cause it's been very useful for me, and jcoyne thought it sounded useful upstream. It is of course off by default. If I remove it, someone else is welcome to try to add one that doesn't have a dependency, although I don't personally think that's the right choice.